pennylane_cirq/simulator_device.py

```diff
@@ -59,11 +59,11 @@ class SimulatorDevice(CirqDevice):
     """
     name = "Cirq Simulator device for PennyLane"
     short_name = "cirq.simulator"
-
-    def __init__(self, wires, shots=1000, analytic=True, qubits=None):
+    # pylint: disable=too-many-arguments
+    def __init__(self, wires, shots=1000, analytic=True, qubits=None, simulator=None):
```

Comment:
Oh, also don't forget to add the new keyword argument to the docstring

Reply:
I would but there wasn't a docstring before. I can add one now tho.

Reply:
There's one above, below the `class :` line :)

Reply:
oh i see